### PR TITLE
Adds a chat command to reload admins in case someone locks themselves out.

### PR DESCRIPTION
**Pull request by silicons**
😄

### CODE REVIEW

**Comment by ninjanomnom** [comment]
image

**Comment by kevinz000** [comment]
image

**Comment by Jordie0608** [comment]
untitled

File: code/modules/admin/chat_commands.dm [hunk 1]
```diff
@@ -81,3 +81,13 @@ GLOBAL_LIST(round_end_notifiees)
...
 	return "I will notify [sender] when the round ends."
+
+/datum/server_tools_command/reload_admins
+	name = "reload_admins"
+	help_text = "Forces the server to reload admins."
+	admin_only = TRUE
+
+/datum/server_tools_command/reload_admins/Run(sender, params)
+	load_admins()
+	log_admin("[sender] reloaded admins via IRC/discord command.")
```
Comment (Cyberboss) [inline code/modules/admin/chat_commands.dm]: `via chat` (regrettably) more providers may be added in the future

**Review by Cyberboss — CHANGES_REQUESTED** [review]

File: code/modules/admin/chat_commands.dm (hunk 1 above)
Comment (ninjanomnom) [reply code/modules/admin/chat_commands.dm]: email

**Review by Cyberboss — APPROVED** [review]

**Comment by ExcessiveUseOfCobblestone** [comment]
Intentionally breaking stuff when normal admins would never do that then adding a "fix" for it seems like the epitome of bloat.

**Comment by Cyberboss** [comment]
reload_admins is still useful to have so we don't need to bug someone with remote when things mess up

**Pull request merged**